frontend/src/StatusIndicator.jsx

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import PropTypes from "prop-types";
```

Comment:
this should be in the `/general/` folder 
Also HeaderAppBar but I can't comment directly on that file

Reply:
Done.